cmake/modules/hunter_windows_utils.cmake

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2013, 2015 Ruslan Baratov
+# All rights reserved.
+
+# CMAKE function to convert windows log filename to 8.3 filename
+function(hunter_windows_to_snf_filename path new_path_variable)
+  set(conversion_cmd
+    "cmd" 
+    "/c" 
+    "for" 
+    "%A" 
+    "in" 
+    ("${path}") 
+    "do" 
+    "@echo"
+    "%~sA"
+	)
```

Comment:
Here is a nice link that explains what happens here: http://stackoverflow.com/a/10227715/2288008. It would be nice to add it as a comment for other developers.